packages/http-header-normalizer/index.js

```diff
@@ -49,7 +53,8 @@ module.exports = (opts) => {
   }
 
   const defaults = {
-    normalizeHeaderKey
+    normalizeHeaderKey,
+    camelCase: false
```

Comment:
can we rename this `camelCase` option to canonical?

It isn't really camel case since it will also have dashes (`-` ) in between